.nix/bindlib.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildDunePackage }:
+
+# We need the very last version "bleeding edge" since previous versions don't use dune.
+
+buildDunePackage rec {
+  pname = "bindlib";
+  version = "5.0.1a";
+
+  minimumOCamlVersion = "4.0.8";
+
+  useDune2 = true;
+
+  src = fetchFromGitHub {
```

Comment:
I'm not very confortable with the duplication of version numbers, etc. that this thing introduces. Right now the `dune-project` files describes the OCaml dependencies completely, the `.opam` file is derived from it. Can't you derive this `.nix` file from `dune-project` too ?

Reply:
There is some tools to generate .nix files from opam files such as [opam2nix](https://github.com/timbertson/opam2nix). Last time i've try to use such tools, it didn't work that well. Moreother, those tools are build by community members, whom might not have all their time to maintain those tools.

There is a major difference between opam and nix package management: for dune/opam you give constraints on the versions numbers, while nix requires version numbers to be explicit. Building a tool that translates dune-project constraints to nix derivations = requires to solve thoses constraints.

Finally, bindlib and unionfind version numbers are not to be understood as "duplication of version numbers", but as appendium to nixpkgs. Fortunally, it occurs that the version supplied is fully compatible with the dependencies of Catala, but it could as well not be the case.